gridmap/job.py

```diff
@@ -116,9 +116,10 @@ class Job(object):
                  'cause_of_death', 'num_resubmits', 'home_address',
                  'log_stderr_fn', 'log_stdout_fn', 'timestamp', 'host_name',
                  'heart_beat', 'track_mem', 'track_cpu', 'interpreting_shell',
-                 'copy_env','par_env')
+                 'copy_env', 'par_env', 'pe', 'cpu', 'h_vmem', 'h_rt', 'resources')
 
     def __init__(self, f, args, kwlist=None, cleanup=True, mem_free="1G",
+                 pe='parallel', cpu=1, h_vmem="4G", h_rt='00:59:00', resources=[],
```

Comment:
Let's set the default for this to be `None` so we don't use a mutable default argument. Then, below check if it's None and, if it is, set it to `[]`. Would you also mind documenting each of the new arguments here in the docstring below, particularly for `resources` since we will want to see an example of how it should look. It looks like it should be something like `["my_custom_resource=1"]`.

Reply:
Maybe it would be best if you just use this PR as a guide for your own changes. I only changed a few lines of code.